blocks/gu_spoverview/templates/spoverview.mustache

```diff
@@ -23,22 +23,22 @@
     <div class="row">
         <div class="assessments-item assessments-submitted col-md-3 col-sm-6 col-xs-12">
             <img src="{{assessments_submitted_icon}}" alt="{{assessments_submitted_str}}" class="assessments-item-icon"/>
-            <h4 class="assessments-item-count color-primary">{{assessments_submitted}}</h4>
+            <h4 class="assessments-item-count color-primary submitted-assignment-count">{{assessments_submitted}}</h4>
```

Comment:
Please do not include updates on this specific file in the commit. (Remove additional classes on h4 elements)